### DISCUSSION TITLE
Display hotkey hints in mod select

### DISCUSSION THREAD

**Discussion by vatei** [ppy/osu#30670] [Ideas]
I think mod select is getting pretty crowded (which is a separate topic in and of itself), and it would be pretty handy to be able to see which hotkey activates each mod, especially in sequential mode.
Thanks for considering!

**Comment by cl8n** [comment]
by "crowded" do you mean it's hard to parse visually or just that there are a lot of mods?

**Reply by vatei** [reply]
Both really, I do feel like there are too much mods nowadays, and that many are redundant and could be merged into each other, with mod settings being a thing. 
On top of that, there’s way too much stuff displayed in mod select that people rarely use. Maybe a way to collapse sections could help? Although I'm not sure that will solve everything, since there might be one mod in a section I'm using a lot, and the others not so much (for exemple I'm mostly using DA in conversion mods, and only using TC in fun mods)
Basically, when using mod select, I wish the stuff I or most people would care about more was shown more prominently, but I don't want the other settings to be so hidden I don't know they exist :p

I hope that makes sense
